readthedocs/templates/base.html

```diff
@@ -271,7 +271,7 @@ <h4>{% trans 'Company' %}</h4>
                     {% endif %}
                   {% endfor %}
                 </select>
-                <input type="submit" value="{% trans 'Change Language' %}" name="submit">
+                <input type="submit" value="{% trans 'Set Language' %}" name="submit">
```

Comment:
Strings already marked for translation shouldn't be changed, that breaks all existing translation. Can you undo this one?

Reply:
What both Crowdin and Weblate can do?

Reply:
> Strings already marked for translation shouldn't be changed, that breaks all existing translation. Can you undo this one?

So you're telling me that ReadTheDocs uses a translation platform (Transifex) that is unable to update the source strings on its own? What is default? What both Crowdin and Weblate can do?

Reply:
I'm aware of your efforts in NodeBB :wink:

Reply:
From a UX perspective, I think "Change language" is more clear.

Reply:
> I'm aware of your efforts in NodeBB 😉

?

Reply:
Done! See 0bcd912